.github/workflows/publish-all-platforms.yml

```
@@ -54,12 +58,14 @@ jobs:
       golang: ${{ !!inputs.golang-package-version }}
       maven: ${{ !!inputs.maven-package-version }}
       kotlin-mpp: ${{ !!inputs.kotlin-mpp-package-version }}
+      react-native: ${{ !!inputs.react-native-package-version }}
       ref: ${{ inputs.ref || github.sha }}
       csharp-package-version: ${{ inputs.csharp-package-version || '0.0.2' }}
       csharp-ref: ${{ inputs.csharp-ref || inputs.ref || github.sha }}
       golang-package-version: ${{ inputs.golang-package-version || '0.0.2' }}
       maven-package-version: ${{ inputs.maven-package-version || '0.0.2' }}
       kotlin-mpp-package-version: ${{ inputs.kotlin-mpp-package-version || '0.0.2' }}
+      react-native-package-version: ${{ inputs.react-native-package-version }}
```

I think it's good to give this a default version, just for testing purposes.
If you run this workflow through the 'push' trigger for example, there are no inputs, so having a default helps with some edge cases.

```suggestion
      react-native-package-version: ${{ inputs.react-native-package-version || '0.0.2' }}
```